ksqldb-functional-tests/src/main/java/io/confluent/ksql/test/tools/TestExecutorUtil.java

```diff
@@ -438,10 +439,13 @@ public ConfiguredKsqlPlan next() {
         return Optional.empty();
       }
 
+      // TODO: cleaner way to handle this?
+      final ConfiguredStatement<?> withFormats =
```

Comment:
Same as above: this is needed or else tests that don't explicitly specify `KEY_FORMAT` in CS/CT statements fail. Feels hacky to add the injector here but I can't think of a better solution (besides requiring explicit `KEY_FORMAT` in all tests, which doesn't seem right).

Reply:
See aboive.